Week1/homework/js-exercises/Exercise 2/errorDebugging.js

```diff
@@ -0,0 +1,2 @@
+'use strict'
+console.log("I'm awesome")
```

Comment:
This is one way to solve this issue. But i'm gonna give you another challenge. What if we have decided we will only use single quotes for strings...
Can you come up with another solution in that case?